engine/api/authentication/gitlab/gitlab.go

```diff
@@ -100,6 +100,10 @@ func (d authDriver) GetUserInfo(ctx context.Context, req sdk.AuthConsumerSigninR
 	}
 
 	c := gitlab.NewOAuthClient(http.DefaultClient, t.AccessToken)
+	err = c.SetBaseURL(d.url)
```

Comment:
one line error check

Reply:
Hello,
Newbie to golang dev.
Do you prefer ?
```golang
	if err := c.SetBaseURL(d.url); err != nil {
		return info, sdk.WrapError(err, "invalid gitlab url")
	}
```

Reply:
Done

Reply:
Done